logstash-core/src/main/java/org/logstash/ObjectMappers.java

```diff
@@ -52,12 +53,27 @@
 import org.jruby.RubySymbol;
 import org.jruby.ext.bigdecimal.RubyBigDecimal;
 import org.logstash.ext.JrubyTimestampExtLibrary;
+import org.logstash.jackson.StreamReadConstraintsUtil;
 import org.logstash.log.RubyBasicObjectSerializer;
 
 public final class ObjectMappers {
 
     static final String RUBY_SERIALIZERS_MODULE_ID = "RubySerializers";
 
+    static final StreamReadConstraintsUtil CONFIGURED_STREAM_READ_CONSTRAINTS;
+
+    static {
+        // The StreamReadConstraintsUtil needs to load the configured constraints from system
+        // properties and apply them _statically_, before any object mappers are initialized.
+        CONFIGURED_STREAM_READ_CONSTRAINTS = StreamReadConstraintsUtil.fromSystemProperties();
+        CONFIGURED_STREAM_READ_CONSTRAINTS.applyAsGlobalDefault();
+    }
+
+    static StreamReadConstraintsUtil getConfiguredStreamReadConstraints() {
```

Comment:
```suggestion
    public static StreamReadConstraintsUtil getConfiguredStreamReadConstraints() {
```

Should be public to allow access from Java

Reply:
yeh, caught in [16403d0](https://github.com/elastic/logstash/pull/16719/commits/16403d08b0d2794e3033d520490bd6ec2fc7d5d7).